common/main/java/com/couchbase/lite/Result.java

```diff
@@ -569,7 +574,7 @@ private void assertInBounds(int index) {
     }
 
     private void assertOpen() {
-        if (rs.isClosed()) {
+        if (context.getResultSet().isClosed()) {
```

Comment:
Hmmm, is this ever going to be true?  Doesn't the no-op close prevent this?

Reply:
yeah.   good point.